cooperative_lanechange/src/cooperative_lanechange.cpp

```diff
@@ -61,6 +61,8 @@ namespace cooperative_lanechange
         twist_sub_ = nh_->subscribe("current_velocity", 1, &CooperativeLaneChangePlugin::twist_cd, this);
         incoming_mobility_response_ = nh_->subscribe("incoming_mobility_response", 1 , &CooperativeLaneChangePlugin::mobilityresponse_cb, this);
         
+        georeference_sub_ = nh_->subscribe("georeference", 1, &CooperativeLaneChangePlugin::georeference_callback, this);
```

Comment:
General question: Are these georeference changes intended to be a part of this changeset specifically? Or are these from another branch which is not yet merged into develop?

Reply:
After merging in Mike's branch (https://github.com/usdot-fhwa-stol/carma-platform/pull/1315), I added this specific line to this changeset since it was required for route generation, but was missing in Mike's original branch. Afterwards, the line was also added to Mike's branch and merged into develop.

All other georeference changes are from Mike's branch (not specific to this changeset) and have been merged into develop.